static/js/common/Tooltip.jsx

```diff
@@ -0,0 +1,184 @@
+import React, { useEffect, useMemo, useRef } from 'react';
+import ReactDOM from 'react-dom';
+import {
+  useFloating,
+  autoUpdate,
+  offset,
+  flip,
+  shift,
+  arrow as arrowMiddleware,
+} from '@floating-ui/react-dom';
+import Button from './Button';
+import { InterfaceText } from '../Misc';
+import Sefaria from '../sefaria/sefaria';
+
+// Configuration constants (kept consistent with existing ModuleSwitcherTooltip)
+const PLACEMENT_CONFIG = {
+  desktop: 'bottom',
+  mobileHebrew: 'right',
+  mobileEnglish: 'left',
+};
+const TOOLTIP_OFFSET = 10;
+const getTooltipShiftPadding = (isMobile) => (isMobile ? 10 : 45);
```

Comment:
device doesn't change so it can be static:
```suggestion
const TOOLTIP_SHIFT_PADDING = Sefaria.multiPanel ? 45 : 10;
```